src/HonzaBotner.Discord.Services/Commands/BotCommands.cs

```diff
@@ -1,131 +1,78 @@
-﻿using System;
-using System.Reflection;
+﻿using System.Reflection;
 using System.Threading.Tasks;
-using DSharpPlus.CommandsNext;
-using DSharpPlus.CommandsNext.Attributes;
 using DSharpPlus.Entities;
-using HonzaBotner.Discord.Services.Attributes;
+using DSharpPlus.SlashCommands;
 using HonzaBotner.Discord.Services.Options;
-using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 
 namespace HonzaBotner.Discord.Services.Commands;
 
-[Group("bot")]
-[Description("Commands to get you info about Discord bot and other things.")]
-public class BotCommands : BaseCommandModule
+public class BotCommands : ApplicationCommandModule
 {
-    private readonly ILogger<BotCommands> _logger;
```

Comment:
I am not against this change, but please be sure to have at least some logging in the commands so we can fine-tune it.

Reply:
True, RN logging is implemented only at the top level, posting errors to discord. I will implement more logging where needed